style.css

```diff
@@ -0,0 +1,488 @@
+.title-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    flex-direction: column;
+}
+
+.articletitle {
+    font-size: 40px;
+    max-width: 50%;
+    font-style: italic;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    margin-left: -150px;
+}
+
+.title-image {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.img-title {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: auto;
+}
+
+.img-title img {
+    width: 80%;
+    height: auto;
+}
+
+.label-image {
+    height: auto;
+    width: 80%;
+    margin-top: 7px;
+}
+
+.label-image .text-label {
+    width: 75%;
+    font-size: 15px;
+    color: #666666;
+}
+
+.article-text {
+    font-size: 20px;
+    color: #333333;
+    margin: auto;
+    width: 40%;
+    text-align: justify;
+    line-height: 1.5;
+    margin-bottom: 15px;
+}
+
+
+.article-writer {
+    font-size: 15px;
+    color: #333333;
+    margin: auto;
+    margin-top: 30px;
+    width: 40%;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+.article-writer a {
+    text-decoration: none;
+    font-weight: bold;
+    color: #333333;
+}
+
+.social-media {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-size: 14px;
+    font-family:Helvetica, sans-serif;
+    color: #333333;
+    margin: auto;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.social-icon {
+    width: 30px;
+    height: auto;
+
+}
+
+.story-meta {
+    width: 70%;
+    margin: auto;
+}
+
+.imagefloat {
+    /* position: relative; */
+    margin: auto;
+    width: 70%;
+    text-align: justify;
+}
+
+.imagefloat span {
+    font-weight: bold;
+}
+
+.imagefloat img {
+    width: 100%;
+
+}
+
+.wrapper {
+    margin-top: -75%;
+    display: flex;
+    flex-direction: column;
+
+
+}
+
+.first {
+    margin-bottom: 20%;
+}
+
+
+#g-ai1 {
+    float: left;
+    margin-right: 9%;
```

Comment:
`g-ai1` through `g-ai4` have the same style properties, we can remove alot of duplicate code with a class having the same properties, and give custom style properties to those who need a little change. Also `g-ai5` through `g-ai8`.